tests/e2e/packages/verify_packages_install_deb.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+# This test installs debian packages that are a result of the CI and release builds.
+# It runs '... --version' commands to verify that the binaries are correctly installed
+# and finally uninstalls the packages.
+# For the release packages it tests all versions in the support skew.
+
+LINE_SEPARATOR="*************************************************"
+echo "$LINE_SEPARATOR"
+
+PACKAGES_TO_TEST_K8S="kubectl kubelet kubeadm"
+PACKAGES_TO_TEST_EXT="cri-tools kubernetes-cni"
+
+PACKAGES_TO_TEST="$PACKAGES_TO_TEST_EXT $PACKAGES_TO_TEST_K8S"
+
+# harcode the LSB release to "xenial" instead of calling "lsb_release -c -s"
+LSB_RELEASE="xenial"
+
+# install prerequisites
+
+echo "* installing prerequisites"
+apt-get update
+apt-get install -y apt-transport-https curl python gnupg
+
+# check if gsutil from the Google Cloud SDK exists and if not install the SDK
+
+if ! `gsutil version > /dev/null`; then
+  curl -sSL https://sdk.cloud.google.com > /tmp/gcl && bash /tmp/gcl --install-dir=~/gcloud --disable-prompts
+  export PATH=$PATH:~/gcloud/google-cloud-sdk/bin
+fi
+gsutil version
+
+# test CI packages
+
+echo "$LINE_SEPARATOR"
+echo "* TESTING CI PACKAGES"
+
+PACKAGE_EXT="deb"
+CI_DIR=/tmp/k8s-ci
+mkdir -p $CI_DIR
+mkdir -p /opt/cni/bin
+CI_VERSION=$(curl -sSL https://dl.k8s.io/ci/latest.txt)
+CI_URL="gs://kubernetes-release-dev/ci/$CI_VERSION-bazel/bin/linux/amd64"
+
+echo "* testing CI version $CI_VERSION"
+
+for CI_PACKAGE in $PACKAGES_TO_TEST; do
+  echo "* downloading package: $CI_URL/$CI_PACKAGE.$PACKAGE_EXT"
+  gsutil cp $CI_URL/$CI_PACKAGE.$PACKAGE_EXT $CI_DIR/$CI_PACKAGE.$PACKAGE_EXT
+  dpkg -i $CI_DIR/$CI_PACKAGE.$PACKAGE_EXT || echo "* ignoring expected 'dpkg -i' result"
+done
+
+echo "* installing"
+
+apt-get install -f -y
+
+echo "* testing binaries"
+
+kubeadm version -o=short
+kubectl version --client=true --short=true
+kubelet --version
+crictl --version
+ls /opt/cni/bin/loopback
+
+echo "* removing packages"
+
+apt-get remove -y --purge $PACKAGES_TO_TEST
+apt-get autoremove -y
+
+rm -rf $CI_DIR
+
+# test stable packages
+# NOTE: MAJOR version bumps are not supported!
+
+echo "$LINE_SEPARATOR"
+echo "* TESTING STABLE PACKAGES"
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+  deb http://apt.kubernetes.io/ kubernetes-$LSB_RELEASE main
```

Comment:
the underlying image is prow jobs will be `debian:stretch`
yet here we basically pass `xenial` (ubuntu 16.04) and install based on that.